src/main/java/com/stripe/model/issuing/Cardholder.java

```diff
@@ -24,6 +25,9 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public class Cardholder extends ApiResource implements HasId, MetadataStore<Cardholder> {
+  @SerializedName("authorization_controls")
+  AuthorizationControls authorizationControls;
+
```

Comment:
While this is merged, it's in theory gated and coming with a new API version I think so we should consider removing for a few days

Reply:
We agreed now that this is okay